### DISCUSSION TITLE
SSL Certificate is expired on Oct 14, 2021

### DISCUSSION THREAD

**Discussion by trinhdh** [NG-ZORRO/ng-zorro-antd#7005] [General] [answered]
Hey guys,
The SSL Certificate of ng.ant.design is expired today Oct 14, 2021. Can someone update ? Thank you

![Screen Shot 2021-10-14 at 9 00 08 PM](https://user-images.githubusercontent.com/3796012/137415797-d6b8a182-0e93-4a61-a1ae-affef5af3f6a.png)
.

**Comment by vthinkxie** [comment, marked as answer]
it should work now, thanks for your feedback
